src/routes/(admin)/questionbank/+page.svelte

```
@@ -197,20 +198,23 @@
 	infoDescription="Question Bank allows you to create, manage, and categorize questions that can be reused in multiple tests. Use filters to quickly find and organize questions by topic or difficulty."
 >
 	{#snippet headerActions()}
-		{#if !noQuestionCreatedYet}
-			{#if canCreate(data.user, 'question')}
-				<a href="/questionbank/single-question/add/new"
-					><Button class="font-bold" variant="outline"><Plus />Create a Question</Button></a
-				>
-				<a href="/questionbank/import"><Button class="font-bold"><Plus />Bulk Upload</Button></a>
-			{/if}
+		{#if canCreate(data.user, 'question')}
+			<a href="/questionbank/single-question/add/new"
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)